android/README.md

```diff
@@ -138,10 +149,11 @@ You can look at the [default TFLite backend implementation](../mobile_back_tflit
 The following steps are required to add your backend:
 
 ### Unified app changes
+
 In java/org/mlperf/inference/BUILD, add the following replacing
 _vendor_ with the actual vendor/backend name:
 
-```
+```text
```

Comment:
Please, use `python` annotation for bazel configs in documentation. Bazel uses Starlark, which is is a subset of python.

Reply:
Done